test/unit/react-imgix.test.jsx

```diff
@@ -41,23 +41,10 @@ const makeBackgroundWithBounds = (bounds) => (props) => (
 
 const src = "http://domain.imgix.net/image.jpg";
 let sut;
-let oldConsole, log;
-
-beforeEach(() => {
-  oldConsole = global.console;
-  delete console.log;
-  console.error = console.log;
-  log = console.log.bind(console);
-});
-afterEach(() => {
-  global.console = oldConsole;
-});
-
-function sleep(ms) {
-  return new Promise((resolve) => {
-    setTimeout(resolve, ms);
-  });
-}
+// TODO(luis): do we need to mock error? There's a react error logging that
+// might make the test-suite fail on CI.
+// jest.spyOn(global.console, 'error').mockImplementation((error) => { console.log(error) })
```

Comment:
Looks like there is a (possibly updated) TODO here?

Reply:
Considering CI hasn't failed (yet) because of this unhandled error in the test suite I think this TODO is safe to remove.